Algorithm.Framework/Alphas/MacdAlphaModel.cs

```diff
@@ -130,7 +130,7 @@ public override void OnSecuritiesChanged(QCAlgorithm algorithm, SecurityChanges
             }
         }
 
-        class SymbolData
+        public class SymbolData
```

Comment:
As the dictionary we want `_symbolData` is protected if the type of its values is also protected we won't be able to return it, because we would be returning a less accessible type than the method itself in `TestMacdAlphaModel.cs` https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs0050?f1url=%3FappId%3Droslyn%26k%3Dk(CS0050)